.github/workflows/scripts/slack_gem_notifications/notifications_methods.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+require 'time'
```

Comment:
What happens if you don't require 'time'?

Reply:
My tests failed without requiring Time. I didn't understand this as I thought it was an included Ruby Class already.